librz/include/rz_core.h

```diff
@@ -836,8 +859,12 @@ RZ_API int rz_core_disasm_pdi(RzCore *core, int nb_opcodes, int nb_bytes, int fm
 RZ_API int rz_core_disasm_pde(RzCore *core, int nb_opcodes, int mode);
 RZ_API RZ_OWN char *rz_core_disasm_instruction(RzCore *core, ut64 addr, ut64 reladdr, RZ_NULLABLE RzAnalysisFunction *fcn, bool color);
 RZ_API bool rz_core_print_function_disasm_json(RzCore *core, RzAnalysisFunction *fcn, PJ *pj);
+RZ_API void rz_core_asm_bb_middle(RZ_NONNULL RzCore *core, ut64 at, RZ_INOUT RZ_NONNULL int *oplen, RZ_NONNULL int *ret);
```

Comment:
is this part of a separate fix? I would move that in a separate PR as this is already quite big and it involves a lot of changes.

Reply:
But there is no problem, if just `aoj` `aOj` `a8` calls `rz_core_analysis_bytes`.

Just breaks the tests when I rewrite `rz_core_print_disasm_json` with `rz_core_analysis_bytes`.